test/test_xpu.py

```diff
@@ -312,6 +312,11 @@ def test_serialization_array_with_empty(self):
 
 
 class TestXpuAutocast(TestCase):
+    # These operators are not implemented on XPU backend and we can NOT fall back
+    # them to CPU. So we have to skip them at this moment.
+    # TODO: remove these operators from skip list when they are implemented on XPU backend.
+    skip_list = ["gru_cell"]
```

Comment:
`_thnn_fused_gru_cell` has not a CPU implementation.

Reply:
_thnn_fused_gru_cell has not XPU nor CPU implementation.